pyomo/solvers/plugins/solvers/GAMS.py

```diff
@@ -43,8 +43,7 @@ class GAMSSolver(pyomo.util.plugin.Plugin):
 
     Pass solver_io keyword arg to SolverFactory to choose solver mode:
         solver_io='direct' or 'python' to use GAMS Python API
-            Requires installation, visit this url for help:
-            https://www.gams.com/latest/docs/apis/examples_python/index.html
+            Requires installation, visit online docs for help.
```

Comment:
visit online docs at `http://www.gams.com`? Otherwise they might look at Pyomo online docs. Same with the docstring under `solve`.

Reply:
The correct url at this point would be `https://www.gams.com/latest/docs/API_PY_TUTORIAL.html`, so perhaps `https://www.gams.com/latest/docs/` would be a little more desciptive but still defend against changes. I'm also just not sure how much to expect them to change it again.

Reply:
Ultimately, that's up to the GAMS folks, but as long as our users get a good general sense of where to go, I think that it's sufficient.